Chummer/Backend/Uniques/AIProgram.cs

```diff
@@ -73,6 +74,11 @@ public void Create(XmlNode objXmlProgramNode, string strExtra = "", bool boolCan
             objXmlProgramNode.TryGetStringFieldQuickly("page", ref _strPage);
             if (!objXmlProgramNode.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlProgramNode.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
+
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlProgramNode.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific AI programs?